src/1_create_canonical.jl

```diff
@@ -113,8 +122,14 @@ output_features .= ifelse.(ismissing.(output_features), "NA", output_features)
 string_cols = contains.(string.(typeof.(eachcol(output_features))), "String")
 output_features[!, contains.(string.(typeof.(eachcol(output_features))), "String")] .= String.(output_features[:, string_cols])
 
-# Correct incorrect GBRMPA_ID to match cots_priority data
-output_features.GBRMPA_ID = ifelse.((output_features.GBRMPA_ID .== "20198"), "20-198", output_features.GBRMPA_ID)
+# Correct incorrect RME_GBRMPA_ID to match RME v1.0.33 data
+output_features.RME_GBRMPA_ID = ifelse.((output_features.RME_GBRMPA_ID .== "20198"), "20-198", output_features.RME_GBRMPA_ID)
```

Comment:
Sorry, why aren't we using the RME id list directly? Shouldn't `RME_GBRMPA_ID` just be a copy of the first column in the `id_list`?